docs/installation.rst

```diff
@@ -1,15 +1,17 @@
 Installation
 ============
 
+.. contents::
+   :local:
+
 Requirements
 ************
 
-- In order to build images, you need to have either ``podman`` or ``docker``
-  installed as well as the ``ansible-builder`` Python package.
-  The ``--container-runtime`` option needs to correspond to the Podman/Docker
-  executable you intend to use.
-- ``ansible-builder`` version ``1.1.0`` requires Python ``3.8`` or higher.
-
+- To build images, you must install either ``podman`` or ``docker``
+  as well as the ``ansible-builder`` Python package.
+- The ``--container-runtime`` option needs to correspond to the Podman/Docker
+  executable you intend to use (where is this set?).
+- ``ansible-builder`` version ``3.x`` requires Python ``???`` or higher.
```

Comment:
```suggestion
- ``ansible-builder`` version ``3.x`` requires Python ``3.8`` or higher.
```

Reply:
wow, 1.1 and 3.x have the same Python version requirement?

Reply:
I'm going to assume @Akasurde is correct.

Reply:
3.9 is actually the minimum for `ansible-builder 3.0`.